src/components/stepProgressBar/StepProgressBar.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { range } from 'underscore';
+import * as classNames from 'classnames';
+
+export interface IStepProgressBarProps {
+  /**
+   * A positive integer above zero
+   */
+  numberOfSteps: number;
+  /**
+   * The 0-based index of the step currently in completion by the user
+   */
+  currentStep: number;
+  className?: string[] | string;
+}
+
+export const StepProgressBar = (props: IStepProgressBarProps) => {
+    const { numberOfSteps, currentStep } = props;
+    const stepWidth = Math.floor(100 / numberOfSteps);
+    const stepProgressBarSteps = range(numberOfSteps).map((stepNumber: number) => (
+      <div
+        className={classNames('step-progress-bar', {
+          'step-progress-bar-done': stepNumber < currentStep,
+          'step-progress-bar-doing': stepNumber === currentStep,
+          'step-progress-bar-to-do': stepNumber > currentStep,
+        })}
+        style={{
```

Comment:
J'ai l'impression que ce serait faisable directement en CSS sans avoir à faire de math. ça lookrait plus clean aussi. Possible?

Reply:
let me check, might be possible yes

Reply:
done by adding `.full-content-x` to each div